agent/config/config.go

```diff
@@ -434,6 +445,25 @@ func getTaskCPUMemLimitEnabled() Conditional {
 	return taskCPUMemLimitEnabled
 }
 
+func getTaskMetatadataRpsLimits() (int, int) {
```

Comment:
maybe i missed an earlier conversation. why don't we want two separate env variables for this? `ECS_TASK_METADATA_RPS_LIMIT` and `ECS_TASK_METADATA_RPS_BURST_LIMIT`?

Reply:
Since steady state and burst throttles both are needed and related, it would be convenient for customers to have a single environment variable.

Reply:
hm i don't feel strongly either way, my concern was mostly if it would add extra overhead of enforcing ordering of the throttle limits (for the customer) and extra validation code (for us). but what you said about "needed and related" also makes sense. okay thanks.